bin/main.rb

```diff
@@ -33,7 +37,12 @@ def move(index, current_player)
   end
 
   def move_board(idx)
-    @board[idx] = 'X'
+    if @current_player == @name[0] 
+      sign = 'X'
+    elsif @current_player == @name[1]
+      sign = 'O'
+    end
+    @board[idx] = sign
   end
```

Comment:
If you had set the player object with name and symbol, all that is needed here would be to call current_player.symbol this assumes that you also create a method in the game class that automatically switches between player 1 and 2 after each move